ports/boost-certify/vcpkg.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "boost-certify",
+  "version-string": "2020-11-14",
+  "description": "Platform-specific TLS keystore abstraction for use with Boost.ASIO and OpenSSL.",
+  "homepage": "https://github.com/djarek/certify",
+  "documentation": "https://djarek.github.io/certify/",
+  "dependencies" : [

```

```suggestion
  "dependencies": [
```
The formatting of the file was wrong. See [here](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#manifests) how to format.

---

Have fixed the formatting (and run the "vcpkg format-manifest" command).